cmd/vgmanager/vgmanager.go

```diff
@@ -132,3 +151,52 @@ func run(_ *cobra.Command, _ []string, opts *Options) error {
 
 	return nil
 }
+
+// addTagToVGs adds a lvms tag to the existing volume groups. This is a temporary logic that should be removed in v4.16.
+func addTagToVGs(ctx context.Context, c client.Client, lvm *lvm.HostLVM, nodeName string, namespace string, logger logr.Logger) error {
```

Comment:
if this fails, lvms will crash. is this desired behavior or should we simply continue?

Reply:
Good point. We should continue in my opinion.